solutions/usaco-920.mdx

```diff
@@ -9,6 +9,61 @@ author: Sofia Yang
 
 <LanguageSection>
 
+<CPPSection>
+	
+```cpp
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+using namespace std;
+ 
+int N, M, answer;
+int L[100001];
+vector<int> S_nbrs[100001], D_nbrs[100001];
```

Comment:
magic numbers are bad
put this in a constant like `MAX_N = 1e5` or smth
then when declaring these arrays do it like `int arr[MAX_N + 1]`

Reply:
also literally no need for `answer`, `N`, and `M` to be global